src/Providers/STSimMapProvider.cs

```diff
@@ -1409,5 +1427,21 @@ private static bool CreateAgeTypeColorMap(Project project, string colorMapFilena
 
             return true;
         }
+
+        private static bool DatasheetHasRows(Scenario s, string datasheetName)
```

Comment:
Wondering if these functions should be moved to a Shared location so that the ChartsProvider can use them eventually too? Maybe ok to leave for now though.

Reply:
Agreed that they should be reused. But I think we are more likely to remember to reuse it if we leave it next to `ShouldShowMultiResolutionCriteriaNodes` since we'll probably look back at this code when we implement the same in ChartProvider.

Reply:
Sorry, I meant move all the new functions to a shared class (not just the `DatasheetHasRows()`) since they will be reused by the ChartProvider!

Reply:
I'm just going to merge now though since I think it will be a minor thing we can refactor once we get to the ChartProvider